client/js/libs/handlebars/colorClass.js

```diff
@@ -0,0 +1,11 @@
+Handlebars.registerHelper(
+	// Generates a string from "color-1" to "color-64" based on an input string
+	"colorClass", function(str) {
+		var hash = 1;
+		for (i = 0; i < str.length; i++) {
+			hash += str.charCodeAt(i);
+		}
+
+		return "color-" + hash % 64;
+	}
+);
```

Comment:
This can definitely be improved with all sorts of crazy hashing. However, considering the end goal, I don't think having a not-perfectly-random solution is not a big deal.

Also, I didn't cache classes as we currently do. I don't think this is going to affect performance though. A fast hashing function (fast over randomness quality) is probably the way to go anyway.